### PR TITLE
Add SUMMARY.md

### PR DESCRIPTION
**Pull request by scimas**
Summarizes all available libraries in a single file for easy browsing. Mentions the file in README.md.

Fix: makes kOS documentation link https in the README.md.

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+### Maths and Statistics
+1. lib_running_average_filter.ks : gives a running average of a given list of 
+   numbers.
+
+### Navigation
+1. lib_navball.ks : provides various state values of a vessel like compass 
+   heading, pitch, roll etc.
+2. lib_pid.ks : provides functions for creating a PID controller. This 
+   functionality has been superseded by kOS buil-in `PIDLoop()`.
```
[the remark this answers is not in the document]
Comment (scimas) [reply SUMMARY.md]: Done

File: SUMMARY.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+This repository is created from libraries submitted by multiple kOS users. All 
+contributions are accepted as long as they satisfy the set requirements. As 
+such there may be multiple libraries that provide the same functionality but 
+take different approaches to it.
+
+This is an attempt at summarising and categorising the libraries by their use 
+cases.
+
+### Atmospheric Navigation
+1. lib_circle_nav.ks : provides functions to help with great circle / geodesic
```
[the remark this answers is not in the document]
Comment (scimas) [reply SUMMARY.md]: Done

**Comment by scimas** [comment]
Recheck please. `lib_raw_user_input.ks` and `lib_seven_seg.ks` were already there, `lib_exec.ks` has now been added.

**Comment by nuggreat** [comment]
right `lib_raw_user_input.ks` and `lib_seven_seg.ks` where there but there is still a missing file `lib_number_dialog.ks`

**Comment by scimas** [comment]
Thanks, and fixed.

**Pull request merged**